components/Nav/Nav.tsx

```diff
@@ -139,11 +139,11 @@ const Nav = ({ session }: { session: Session | null }) => {
                       <Link
```

Comment:
@CarolinaCobo looks like there is no gap between the bell icon and theme toggle, let add that too. Everything else looks good.